src/jit/compiler.h

```diff
@@ -1529,6 +1529,7 @@ class Compiler
     friend class TempDsc;
     friend class LIR;
     friend class ObjectAllocator;
+    friend struct GenTree;
```

Comment:
This was "required" for accessing the HWIntrinsic table lookup methods.

This could also be resolved by making those methods public, but we should probably just move them off of compiler and make them public static members of HWIntrinsicInfo, since they don't really have anything to do with the compiler.

Reply:
> make them public static members of HWIntrinsicInfo

This would very helpful change.